app/models/comment.rb

```diff
@@ -2,9 +2,21 @@ class Comment < ApplicationRecord
   belongs_to :post
   validates_presence_of :body
 
+  def self.best(comments)
+    comments.sort { |x,y| determine_sentiment(y) <=> determine_sentiment(x) }.first
+  end
+
+  def self.rank_order(comments)
```

Comment:
Did you consider extracting this method and `self.best` to the `SentimentAnalysisService`?